### PR TITLE
react-select: add tests

### PR DESCRIPTION
**Pull request by smhigley**
Adds vr tests and additional component tests to the react-select package.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `183.617 kB`<br />`50.924 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `33.988 kB`<br />`11.108 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |   `18.252 kB`<br />`6.511 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/b1fcc4e01caf93fe1f9da68b60563f986387c9af'>b1fcc4e01caf93fe1f9da68b60563f986387c9af</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23374/perf-test-react-components/Avatar-mount_base.html">1197</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23374/perf-test-react-components/Avatar-mount.html">1233</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23374/perf-test-react-components/Button-mount_base.html">767</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23374/perf-test-react-components/Button-mount.html">751</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23374/perf-test-react-components/FluentProvider-mount_base.html">2230</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23374/perf-test-react-components/FluentProvider-mount.html">2218</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23374/perf-test-react-components/FluentProviderWithTheme-mount_base.html">398</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23374/perf-test-react-components/FluentProviderWithTheme-mount.html">393</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23374/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">329</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23374/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">329</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23374/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">426</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23374/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">425</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23374/perf-test-react-components/MakeStyles-mount_base.html">1868</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23374/perf-test-react-components/MakeStyles-mount.html">1920</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by micahgodbolt** [comment]
screener changes approved, but it seems it errored out due to a missing selector 

image

**Comment by smhigley** [comment]
@micahgodbolt looks like Screener doesn't like me trying to focus a disabled element :D. I meant to use that test to verify that focusing didn't do anything, but I'll just remove it.

File: apps/vr-tests-react-components/src/stories/Select.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      <Select appearance="filled-lighter">
+        <option>text</option>
+      </Select>
+    </div>
+  ));
+
+// Non-interactive stories
+storiesOf('Select Converged', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  // note: due to reused "Input Converged" story ID, TestWrapperDecoratorFixedWidth is also reused
```
Comment (khmakoto) [inline apps/vr-tests-react-components/src/stories/Select.stories.tsx:51]:
```suggestion
  // note: due to reused "Select Converged" story ID, TestWrapperDecoratorFixedWidth is also reused
```

**Review by khmakoto — APPROVED** [review]

**Pull request merged**